### PR TITLE
3268 - DB Connection Parameter change to crunchy

### PR DESCRIPTION
**Pull request by guru-aot**
- Applications DB connection parameters are changed from patroni to crunchy. 
- DB_SERVICE_NAME is moved as part of crunchy secret, so removed it from makefile parameters.

### CODE REVIEW

File: devops/openshift/api-deploy.yml [hunk 1]
```diff
@@ -323,13 +326,13 @@ parameters:
   - name: PORT
     required: true
   - name: DB_SERVICE
-    value: patroni-master
+    value: pgbouncer-host
```
Comment (dheepak-aot) [inline devops/openshift/api-deploy.yml:329]: I see. This is now an auto generated host name coming from secrets. 
image

File: devops/openshift/api-deploy.yml [hunk 2]
```diff
@@ -58,6 +56,11 @@ objects:
                   value: "${MAXIMUM_IDLE_TIME_FOR_WARNING_INSTITUTION}"
                 - name: MAXIMUM_IDLE_TIME_FOR_WARNING_AEST
                   value: "${MAXIMUM_IDLE_TIME_FOR_WARNING_AEST}"
+                - name: POSTGRES_HOST
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${DB_SERVICE}
```
Comment (dheepak-aot) [inline devops/openshift/api-deploy.yml:62]: As per our naming convention of secret keys, this should be `DB_SERVICE_KEY`

File: devops/openshift/api-deploy.yml (hunk 1 above)
Comment (guru-aot) [reply devops/openshift/api-deploy.yml:329]: yes

File: devops/openshift/api-deploy.yml [hunk 3]
```diff
@@ -322,14 +325,14 @@ parameters:
...
-  - name: DB_SERVICE
-    value: patroni-master
+  - name: DB_SERVICE_KEY
+    value: pgbouncer-host
   - name: DB_SECRET_NAME
-    value: patroni-creds
+    value: simsdb-pguser-app-database-user
   - name: DB_USERNAME_KEY
-    value: database-user
+    value: user
```
Comment (dheepak-aot) [inline devops/openshift/api-deploy.yml:333]: The API and DB Migrations are are using the same user here. I believe the idea for apps(api, workers etc..) is to use non privileged user right?? 

Or is this part going to be handled later?

**Comment by dheepak-aot** [comment]
Good work @guru-aot . Just an outstanding question.

File: devops/openshift/api-deploy.yml (hunk 3 above)
Comment (guru-aot) [reply devops/openshift/api-deploy.yml:333]: We have different user, if you check the SECRET_NAME_KEY both are pointing to 2 different secrets, but the DB_USERNAME_KEY looks the same 
image
image

**Review by dheepak-aot — APPROVED** [review]
Thanks for the clarification. Looks good.

**Pull request closed without merging**